spec/integration/store/GlobalHeader.spec.js

```diff
@@ -0,0 +1,99 @@
+import { createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+import { ipcMain, ipcRenderer } from '~/spec/mock/electron'
+import GlobalHeader from '~/src/renderer/store/GlobalHeader'
+import router from '~/spec/mock/router'
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - 'router' is defined but never used. (no-unused-vars)